crates/bevy_input/src/gamepad.rs

```diff
@@ -473,114 +485,386 @@ impl GamepadSettings {
     }
 }
 
-/// Settings for a [`GamepadButton`].
-///
-/// ## Usage
-///
-/// It is used inside of the [`GamepadSettings`] to define the threshold for a gamepad button
```

Comment:
These two sentences are worth preserving :)

Reply:
It felt weird keeping docs on private fields, but I can resurrect that.

Reply:
Brought back and updated that comment.